### DISCUSSION TITLE
ACER Card8a

### DISCUSSION THREAD

**Discussion by kevin0426ys** [njoy/NJOY21#152] [Q&A]
When I was using Njoy21 to deal with thermalized nuclides, ACER's Card 8a could not recognize 1001

[error] Encountered invalid value for iza

1001/

**Comment by whaeck** [comment]
This might be an error in the input verification. can you include the input file you used?

As a workaround to continue your work, I would suggest you use NJOY2016 directly (https://github.com/njoy/NJOY2016) instead of using the NJOY21 wrapper. Building NJOY2016 is done in the same way as NJOY21, except that you do not need to specify a version tag or branch when cloning. You can just use the master branch which is the latest NJOY2016 release.

**Comment by kevin0426ys** [comment]
thanks，My input files work fine in NJOY2016, but not in NJOY21
